core/domain/exp_domain.py

```diff
@@ -81,9 +81,18 @@
 CMD_RENAME_STATE = 'rename_state'
 # This takes an additional 'state_name' parameter.
 CMD_DELETE_STATE = 'delete_state'
+# DEPRECATED: This command is deprecated. Please do not use. The command remains
+# here to support old suggestions.
 # This takes additional 'state_name', 'content_id', 'language_code' and
 # 'content_html' and 'translation_html' parameters.
 CMD_ADD_TRANSLATION = 'add_translation'
```

Comment:
@kevintab95 Can you please create an issue to remove this command after migrating the suggestion with this command to follow the schema of the new command? (And add a TODO comment?)

Reply:
Done!